### PR TITLE
[ARM] `az tag create`: Add the note for handling existing tag in help

### PR DESCRIPTION
**Pull request by CarstenHS**
Devs should be aware thet az tag create wipes existing tags and informed by a potential solution.

**Description**

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
resource

File: src/azure-cli/azure/cli/command_modules/resource/_help.py [hunk 1]
```diff
@@ -2278,6 +2278,7 @@
 helps['tag create'] = """
 type: command
 short-summary: Create tags on a specific resource.
+note: az tag wipes existing tags. To solve this, one can get and buffer existing tags and add new ones to buffer. Then set buffered tags on resource.
```
Comment (zhoxing-ms) [inline src/azure-cli/azure/cli/command_modules/resource/_help.py:2281]: In fact, there is no `note` entry in help. Please add this part to the end of `long-summary`.
For how to write help, please refer to this doc [authoring_help]( https://github.com/Azure/azure-cli/blob/dev/doc/authoring_help.md)

**Comment by zhoxing-ms** [comment]
@CarstenHS Any update?

**Comment by CarstenHS** [comment]
> @CarstenHS Any update?

Yeah, thanks for your pointer to help.
I've updated to long summary as suggested and rephrased to be more understandable.

File: src/azure-cli/azure/cli/command_modules/resource/_help.py [hunk 2]
```diff
@@ -2278,10 +2278,12 @@
...
```
Comment (zhoxing-ms) [inline src/azure-cli/azure/cli/command_modules/resource/_help.py:2281]:
```suggestion
```

Comment (zhoxing-ms) [reply src/azure-cli/azure/cli/command_modules/resource/_help.py:2281]: https://github.com/Azure/azure-cli/pull/19057#discussion_r684063125 As I mentioned above, this line is no longer needed, so I deleted it~

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**